nssp_demo/post_process.R

```diff
@@ -118,35 +160,34 @@ make_forecast_fig <- function(model_dir) {
   forecast_plot
 }
 
+forecast_fig <- make_forecast_fig(model_dir)
 
-base_dir <- path(here(
+save_plot(
+  filename = path(model_dir, "forecast_plot", ext = "pdf"),
+  plot = forecast_fig,
+  device = cairo_pdf, base_height = 6
+)
+
+
+# Temp code while command line version doesn't work
```

Comment:
Noting that this can be fixed in #66

Reply:
This is dependent on https://github.com/rstudio/renv/pull/2018.